preview/test/PreviewServerTest.scala

```diff
@@ -13,7 +13,7 @@ class PreviewTestSuite extends Suites (
   // These features are tested elsewhere, this is actually just here to ensure that the
   // preview server can start up and serve a page
 
-  "Preview Server" should "be able to serve an article" in goTo("/technology/2014/may/18/de-rosa-idol-bicycle-review-martin-love") { browser =>
-    browser.$("body").text should include ("Debating the pros and cons of each of these materials is the kind of thing that keeps passionate cyclists awake at night")
+  "Preview Server" should "be able to serve an article" in goTo("/music/2015/jan/01/music-awards-for-new-artists-blessing-or-curse") { browser =>
```

Comment:
What happened here, for reference?

Reply:
I am not quite sure, testing locally in the browser worked well and I could preview that article. I tried replacing with several others and they all passed the test. I can dig further if needed, by printing out the hex dump of the response, but that will take some time.

Reply:
I think it's fine, just odd though right!